adli_system.py

```
@@ -0,0 +1,69 @@
+        "adli_system_paths",
+        type=str,
+        help="Path to the system definition file."
+    )
+    
+    parsed_args = args_parser.parse_args(argv[1:])
+    sys_def_file_path = parsed_args.adli_system_paths
+
+    return run(sys_def_file_path)
+
+
+if "__main__" == __name__:
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `__name__ == "__main__"`

(SIM300)